src/components/uploader/js/FileQueueView.js

```diff
@@ -242,6 +248,7 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
 
         // update the click event and the styling for the file delete button
         var removeRowBtn = that.locate("fileIconBtn", row);
+        removeRowBtn.prop("disabled", true);
```

Comment:
Is this new line necessary? The `disabled` attributes on remove buttons have already been set to `true` at `prepareForUpload()`.

Also, it would be nice to sync up variable names used in different functions for same elements, for instance, vars used at line 244, 250 can be synced up with those at line 226, 227.